### ISSUE TITLE
ERA001 false positive on "# pytest: foo=bar"

### ISSUE THREAD

**Issue by qthequartermasterman** [astral-sh/ruff#28675]
### Summary

ERA001 (commented-out-code) incorrectly flags a pytest-prefixed comment as commented-out code.

I discovered this accidentally after typing pytest instead of pylint in a disable comment. Although the comment is invalid, it does not appear to be valid Python code and therefore should not be identified as commented-out code.

This only happens when pytest is imported in a file.

```python
import pytest

# pytest: foo=bar
```

Gives the error message:

```
commented-out-code: Found commented-out code
 --> ruff_lint_mre.py:3:1
  |
1 | import pytest
2 |
3 | # pytest: foo=bar
  | ^^^^^^^^^^^^^^^^^
4 |
help: Remove commented-out code
```

### Version

ruff 0.16.7

**Comment by ntBre** [comment]
Hmm, I think this is working as intended. These are both valid annotated assignments:

```python
pytest: foo = bar
x: int = 1
```

and it sounds like it actually drew your attention to the invalid `pylint` suppression!
